saos-webapp/src/main/resources/saos.default.properties

```diff
@@ -96,4 +96,7 @@ piwik.siteId = 0
 
 
 # ehcache configuration file path
-ehcache.configurationFilePath = classpath:ehcache.xml
\ No newline at end of file
+ehcache.configurationFilePath = classpath:ehcache.xml
+
+# domain name
+domain.name = https://saos.org.pl
```

Comment:
Maybe we should have here https://<b>www.</b>saos.org.pl
It would be better to propagate this address because it doesn't need a redirect when invoked.